examples/dependencies/index.js

```diff
@@ -15,3 +15,5 @@
 
 import '@polymer/iron-demo-helpers/demo-snippet.js';
 import '@polymer/paper-button/paper-button.js';
+import '@polymer/paper-slider/paper-slider.js';
+import '@polymer/paper-radio-group/paper-radio-group.js';
```

Comment:
I like using these nice components in our demos, but maybe these should be broken out per dependency as-needed in demos (e.g. only the PBR demo and fuzzer use paper-button)

Reply:
:+1: